scripts/biomedical/mesh/format_mesh.py

```diff
@@ -0,0 +1,278 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+'''
+Author: Suhana Bedi
+Date: 09/17/2021
+Name: format_mesh.py
+Description: converts nested .xml to .csv and further breaks down the csv into 
+four different csvs, each describing relations between terms, qualifiers, descriptors and
+concepts.
+@file_input: input .xml downloaded from NCBI 
+'''
+import sys
+import pandas as pd
+import numpy as np
+from xml.etree.ElementTree import parse
+
+
+def format_mesh_xml(mesh_xml):
```

Comment:
I've not validated the logic of the Python code.  I'm assuming @spiekos will do that :)